templates/scenarios/ts/link-unfurling/linkUnfurlingBot.ts

```diff
@@ -0,0 +1,72 @@
+import {
+  TeamsActivityHandler,
+  CardFactory,
+  TurnContext,
+  MessagingExtensionResponse,
+  AppBasedLinkQuery,
+} from "botbuilder";
+import card from "./card.json";
+
+export class LinkUnfurlingBot extends TeamsActivityHandler {
```

Comment:
I used class name from https://github.com/OfficeDev/Microsoft-Teams-Samples/blob/main/samples/msgext-link-unfurling/nodejs/bots/teamsLinkUnfurlingBot.js, but I'm thinking about removing bot concept in this template. Should we use name such as "LinkUnfurlingApp"?

Reply:
LinkUnfurlingBot is OK for me.

Reply:
Let's use `LinkUnfurlingApp`, Bot isn't a terminology on Outlook.

Reply:
Agree with Junjie, we'd better to use LinkUnfurlingApp